src/bower.json

```diff
@@ -7,5 +7,10 @@
     "leaflet": "^0.7.3",
     "multiple-select": "~1.1.0",
     "ramda": "0.8.0"
+  },
+  "overrides": {
+    "leaflet": {
+      "ignore": true
```

Comment:
I agree that this would be the ideal solution. That _should_ be trivial, but maybe it's not.